web3-react/src/config.ts

```diff
@@ -0,0 +1,24 @@
+// Sets if the example should run locally or on chain
+export enum Chain {
+  POLYGON,
+  MAINNET,
```

Comment:
If we're using polygon and mainnet only, can we drop all other dependencies like `gnosis` etc. from package.json?

Reply:
So gnosis is a type of connector - not sure how it is related to the chain we support?